README.md

```diff
@@ -344,14 +344,16 @@ There are a growing number of tools that are built atop or support Zod natively!
 
 - TypeScript 4.1+!
 - You must enable `strict` mode in your `tsconfig.json`. This is a best practice for all TypeScript projects.
+- Make sure that `strictNullChecks` mode is also enabled, otherwise static typechecking will return all properties as optional!
 
   ```ts
   // tsconfig.json
   {
     // ...
     "compilerOptions": {
       // ...
-      "strict": true
+      "strict": true,
+      "strictNullChecks: true,
```

Comment:
I don't think this line should be part of our setup instructions. It's only the case that you shouldn't set it to `false`, rather than that you need to set it to `true`.